store/store.go

```diff
@@ -89,18 +89,19 @@ func (s *store) Close() error {
 	return s.db.Close()
 }
 
-func (s *store) SaveBlock(height uint32, block *block.Block, cert *certificate.Certificate) {
+func (s *store) SaveBlock(block *block.Block, cert *certificate.Certificate) {
 	s.lk.Lock()
 	defer s.lk.Unlock()
 
+	height := cert.Height()
 	reg := s.blockStore.saveBlock(s.batch, height, block)
 	for i, trx := range block.Transactions() {
 		s.txStore.saveTx(s.batch, trx.ID(), &reg[i])
 	}
 
 	// Save last certificate
-	w := bytes.NewBuffer(make([]byte, 0, 8+cert.SerializeSize()))
-	err := encoding.WriteElements(w, lastStoreVersion, height)
+	w := bytes.NewBuffer(make([]byte, 0, 4+cert.SerializeSize()))
```

Comment:
@themantre 
it would be better if write some comments about what means `4` for capacity of buffer